ext/drmgr/drmgr.c

```diff
@@ -2497,3 +2501,56 @@ drmgr_disable_auto_predication(void *drcontext, instrlist_t *ilist)
     instrlist_set_auto_predicate(ilist, DR_PRED_NONE);
     return true;
 }
+
+/* Reserve space for emulation specific note values */
+static ptr_uint_t note_base;
```

Comment:
The style guide requests that vars and types be at the top of the file or after banner separators.  How about a banner separator for the emulation var + functions?

Reply:
Done. My bad for not paying attention to the style guide.